assembl/views/api2/discussion.py

```diff
@@ -1524,6 +1529,92 @@ def get_participant_time_series_analytics(request):
     return csv_response(rows, format)
 
 
+def convert_to_utf8(rowdict):
+    row = {}
+    for key, value in rowdict.items():
+        row[key.encode('utf-8')] = value.encode('utf-8') if isinstance(value, unicode) else value
+
+    return row
+
+
+@view_config(context=InstanceContext, request_method='GET',
+             ctx_instance_class=Discussion, permission=P_ADMIN_DISC,
+             name="phase1_csv_export")
+def phase1_csv_export(request):
+    """CSV export for phase 1."""
+    language = 'fr'
+    discussion_id = request.context.get_discussion_id()
+    THEMATIC_NAME = u"Nom de la thématique"
```

Comment:
After discussion with Pauline and Roxana, we decided to force the locale in French for the content, for this first version of the admin interface. Does it makes sense to internationalize the headers now? Having the headers in English (based on user language preference?) and the content in French?